src/components/language-selector/_macro.spec.js

```diff
@@ -124,8 +124,9 @@ describe('macro: language-selector', () => {
         it('does not show the current language', () => {
             const $ = cheerio.load(renderComponent('language-selector', EXAMPLE_WITH_THREE_LANGUAGES));
 
-            expect($('.ons-language-links__item:first-child a').text()).toBe('English');
-            expect($('.ons-language-links__item:last-child a').text()).toBe('Polski');
+            // .replace(/\s+/g, ' ') will replace any sequence of whitespace characters (spaces, tabs, newlines) with a single space
+            expect($('.ons-language-links__item:first-child a').text().replace(/\s+/g, ' ').trim()).toBe('Change language to English');
```

Comment:
I think updating the macro and changing
`<span class="ons-u-vh">Change language to{ ' ' }</span>`
to
`<span class="ons-u-vh">Change language to </span>`
should fix this issue